assignment/scripts/function-practice.js

```diff
@@ -14,31 +14,35 @@ console.log('Test - should say "Hello World!"', hello());
 
 // 2. Function to return an personalized hello, using the `name` argument.
 //    for example 'Hello, Jo!', or 'Hello, Stacy!'
-function helloName( name ) {
-  return;
+function helloName(name = 'my friend') {
+  return `Hello, ${name}!`;
 }
 // Remember to call the function to test
+console.log('Calling helloName("Jo")', helloName("Jo"));
+console.log('Calling helloName()', helloName());
 
 
 // 3. Function to add two numbers together & return the result
-function addNumbers( firstNumber ) {
-  // return firstNumber + secondNumber;
+function addNumbers(firstNumber, secondNumber) {
+  return firstNumber + secondNumber;
 }
+console.log('addNumbers(5, 113)', addNumbers(5, 113));
+console.log('addNumbers(-5, 8)', addNumbers(-5, 8));
 
 
 // 4. Function to multiply three numbers & return the result
-function multiplyThree( ){
-
+function multiplyThree(num1, num2, num3) {
+  return num1 * num2 * num3;
 }
-
+console.log('multiplyThree(2, 3, 4)', multiplyThree(2, 3, 4));
 
 // 5. Function that will return true if a number is positive, 
 //    or greater than zero, and false otherwise
-function isPositive( number ) {
-  if ( number > 0 ){
-    return;
+function isPositive(number) {
+  if (number > 0) {
```

Comment:
Since `>` produces a boolean, you could also have just returned the condition directly:

```js
function isPositive(number) {
  return number > 0;
}
```